applications/HDF5Application/custom_io/hdf5_file.cpp

```diff
@@ -209,7 +209,7 @@ bool File::HasPath(const std::string& rPath) const
 {
     KRATOS_TRY;
     // Expects a valid path.
-    KRATOS_ERROR_IF_NOT(Internals::IsPath(rPath)) << "Invalid path: \"" << rPath << "\". Path should only have characters A-Z, a-z, 0-9, \"/\", and \"_\"." << std::endl;
+    KRATOS_ERROR_IF_NOT(Internals::IsPath(rPath)) << "Invalid path: \"" << rPath << "\". Path should start with \"/\" and should only have characters A-Z, a-z, 0-9, \"/\", and \"_\"." << std::endl;
 
     std::vector<std::string> splitted_path = Internals::Split(rPath, '/');
```

Comment:
BTW I think the StringUtilities in the Core implement the same in the meantime

Reply:
https://github.com/KratosMultiphysics/Kratos/blob/9a003679c1a12a159466e6e61a371e1286f379ca/kratos/utilities/string_utilities.h#L89-L97

Reply:
done :)